src/Stripe.net/Services/Customers/CustomerCreateOptions.cs

```diff
@@ -108,9 +108,6 @@ public class CustomerCreateOptions : BaseOptions, IHasMetadata
         [JsonProperty("tax_id_data")]
         public List<CustomerTaxIdDataOptions> TaxIdData { get; set; }
 
-        [JsonProperty("tax_percent")]
```

Comment:
That one is not a minor change, we should be really careful shipping this changed without a clear understanding of how many users on the latest stripe-dotnet major are passing it. Can you investigate this first?

Reply:
I was only able to find 5 requests in the last 2 weeks for anyone using the past 3 majors.

Reply:
Ack so we should remove this one on all resources/params for good in that case :)